src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup.HydraAdapter/Commands.RecoveryServices.Backup.HydraAdapter.csproj

```diff
@@ -70,6 +70,12 @@
   <ItemGroup>
     <Content Include="Resources\Microsoft.Azure.Management.RecoveryServicesBackupManagement.dll" />
   </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\Commands.RecoveryServices.Backup.Helpers\Commands.RecoveryServices.Backup.Helpers.csproj">
+      <Project>{0e1d3f36-e6c8-4764-8c7d-6f9ee537490c}</Project>
+      <Name>Commands.RecoveryServices.Backup.Helpers</Name>
```

Comment:
I misunderstood a file's scope. Moving a helper to another file in HydraAdapter.